sale_commission/wizard/wizard_settle.py

```diff
@@ -47,6 +47,23 @@ def _get_next_period_date(self, agent, current_date):
         else:
             raise exceptions.Warning(_("Settlement period not valid."))
 
+    def _get_settlement_domain(self, agent, company, sett_from, sett_to):
```

Comment:
Instead of the domain, return the record you want, as you might change order or other requirement for getting a different settlement.